### PR TITLE
more on sharing and the right sidebar

### PR DESCRIPTION
**Pull request by EParzefall**
Subject says it all

### CODE REVIEW

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 1]
```diff
@@ -49,39 +49,47 @@ Under `Personal` in the left sidebar you have your private space where you can u
...
+A click on the three vertical dots opens an actions menu. Actions you can performed on a file or folder depend on the file type (video, text, etc.) and on the permissions granted by the sharer.
 
 === Spaces
 
 Spaces are organizational units to sort content and make it available to certain members.
 
 Under `Personal`, every user has a private space, while the `Spaces` area is intended for collaboration, i.e., sharing on a larger scale.
 
-In spaces, editors can do a little more and there's the additional role of manager.
+In apaces, editors can do a little more and there's the additional role of manager.
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_for_users.adoc:60]: sounds strange - maybe a typo?

**Comment by mmattel** [comment]
general note: we should use one way writing "fixme" because:
* in ocis we use `fixme`
* here we are going to use `FIXME`

This makes it hard to keep in mind and find the lines with grep...

File: modules/owncloud_web/pages/web_for_users.adoc (hunk 1 above)
Comment (EParzefall) [reply modules/owncloud_web/pages/web_for_users.adoc:60]: > sounds strange - maybe a typo?

Definitely. I think I wanted to change it to S

File: modules/owncloud_web/pages/web_for_users.adoc [hunk 2]
```diff
@@ -49,39 +49,47 @@ Under `Personal` in the left sidebar you have your private space where you can u
...
+
+A public link also works for external people not registered on the oCIS server. Below the `Share with people` section, provide a name for the link and set an `Expiration date` and a password before hitting btn:[Share].
+For security reasons, the only possible role for unregistered users is `viewer`.
 
-TIP: If files in one of the spaces you are a member of are deleted, they are placed in the trashbin of the respective space so they can be stored there. Go to the `Spaces` overview, click on the vertical three dots in the space representation and select `Deleted Files`. You'll be directed to the space's trashbin where you can delete for real or restore.
+=== Versions
 
+If you modified a document stored in oCIS, you can always restore older versions. A nice feature, but when it comes to collaboration, versioning is particularly important. If you share a document with an editor, you want to be able to see what happened with the document.
 
+// FIXME: Elaborate more when it's working. Is there a diff to come? I hope so.
```
Comment (EParzefall) [inline modules/owncloud_web/pages/web_for_users.adoc:94]:
```suggestion
// fixme: Elaborate more when it's working. Is there a diff to come? I hope so.
```

**Review by mmattel — APPROVED** [review]

**Pull request merged**